packages/projection/src/operators/Mappers/withCIP67.ts

```diff
@@ -38,11 +38,28 @@ export const withCIP67 = unifiedProjectorOperator<WithUtxo, WithCIP67>((evt) =>
       })
     )
     .filter(isNotNil);
+
+  const mintedCip67Assets = [...(evt.block.body[0].body.mint?.keys() ?? [])]
+    .map((assetId): CIP67Asset | null => {
+      const assetName = Cardano.AssetId.getAssetName(assetId);
+      const decoded = Asset.AssetNameLabel.decode(assetName);
+      if (!decoded) return null;
+      return {
+        assetId,
+        assetName,
+        decoded,
+        policyId: Cardano.AssetId.getPolicyId(assetId)
+      };
+    })
+    .filter(isNotNil);
+
   return {
     ...evt,
     cip67: {
-      byAssetId: Object.fromEntries(cip67Assets.map((asset) => [asset.assetId, asset])),
-      byLabel: groupBy(cip67Assets, (e) => e.decoded.label)
+      byAssetId: {
+        ...Object.fromEntries([...cip67Assets, ...mintedCip67Assets].map((asset) => [asset.assetId, asset]))
+      },
+      byLabel: groupBy([...cip67Assets, ...mintedCip67Assets], (e) => e.decoded?.label)
```

Comment:
If an asset is minted, it will also be present in utxo. Original order will set utxo to undefined, so I think that's a bug. Please add unit tests that cover changes in `withCIP67`.
```suggestion
        ...Object.fromEntries([...mintedCip67Assets..., ...cip67Assets].map((asset) => [asset.assetId, asset]))
      },
      byLabel: groupBy([...mintedCip67Assets..., ...cip67Assets], (e) => e.decoded?.label)
```

Reply:
addressed: 55fa7474e813b502fc2c81d9df6043f4a83d575a

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/993#discussion_r1402174184